rgw/v2/tests/s3cmd/configs/test_header_size.yaml

```diff
@@ -0,0 +1,5 @@
+# script: test_header_size.py
+config:
+  user_count: 1
+  bucket_count: 1
```

Comment:
I think script will ignore second user or second bucket if we increase `user_count` or `bucket_count` from 1 to more

Reply:
Made changes